### DISCUSSION TITLE
Bug en codigo base

### DISCUSSION THREAD

**Discussion by waknaudt** [IIC2133-PUC/2024-2#26] [Tarea 0] [answered]
Hola, en el codigo base que se nos entrega hay una parte que dice 
`char color;
fscanf(input_file, "%d %s", &puffle_id, &color);`

Esto esta causando un bug en mi codigo ya que al parecer se almacena mas de un caracter en la direccion &color lo que sobreescribe otras variables de mi programa. Si cambio estas lineas el error desaparece. Se permite cambiar estas lineas?

**Comment by tomastrivino** [comment, marked as answer]
Hola,
Sí. Por supuesto que puedes cambiar el código base a medida que estimes conveniente.
